pgm_learner/DiscreteGraphState[] states # trial data
float64 pvalparam # optional
uint16 indegree # optional
---
pgm_learner/GraphStructure graph # structure of network
